pallene/coder.lua

```diff
@@ -165,8 +171,8 @@ local function pallene_type_tag(typ)
     elseif tag == "types.T.Array"    then return "LUA_TTABLE"
     elseif tag == "types.T.Table"    then return "LUA_TTABLE"
     elseif tag == "types.T.Record"   then return "LUA_TUSERDATA"
-    elseif tag == "types.T.Any"    then error("value is not a tag")
-    else error("impossible")
+    elseif tag == "types.T.Any"      then typedecl.tag_error(tag, "value is not a tag")
```

Comment:
I just noticed that this error message is outdated. In the past, the `any` type used to be called `value` but we forgot to update the error message when that changes.

Could you please change the error message to `"Any" is not a Lua type tag`?

Reply:
will do